CHANGES.md

```diff
@@ -1,5 +1,8 @@
-### 0.4.1 (2019-09-05)
+### 0.5.0 (2019-10-17)
+* Adds option to omit the Ion version marker from text output. (#109)
+* Adds support for timestamps with precision beyond microseconds. (#100)
 
```

Comment:
i'd suggest including the tuple_as_sexp option added by https://github.com/amzn/ion-python/commit/7094dfdfd73683b5a927cc5193279014512980c3 as well

Reply:
Good point, added.